Library/Homebrew/extend/predicable.rb

```diff
@@ -1,7 +1,8 @@
-# typed: true
+# typed: strict
 # frozen_string_literal: true
 
 module Predicable
+  sig { params(attrs: Symbol).void }
   def attr_predicate(*attrs)
     attrs.each do |attr|
```

Comment:
It looks like `attrs` can be an `Enumerable` here.

Reply:
If I'm understanding you, we want the member type here (`Symbol`, though technically `String` would also be permissible), but `attrs` is implicitly an `Array` bc it's being used as a rest parameter: https://sorbet.org/docs/sigs#rest-parameters

Reply:
Ah, gotcha. My mistake!